src/Paket.Core/PaketConfigFiles/LockFile.fs

```diff
@@ -372,7 +372,7 @@ module LockFileParser =
                     | e ->
                         try InstallSettings.Parse(true, "framework: " + settingsPart) // backwards compatible
                         with e2 ->
-                            raise <| AggregateException(sprintf "failed to parse line '%s'" line, e, e2)
+                            raise <| AggregateException(sprintf "failed to parse line '%s'. Message: %s" line e2.Message, e, e2)
```

Comment:
There shouldn't be a need for these changes as the error reporter at the end outputs all inner exception. If there is a need for these kind of changes we failed to propagate error information correctly and we should fix that instead.

Reply:
I think this was my mistake. I didn't read the message top to bottom and so I thought this would be needed,